.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ e2e-kind-test: &e2e-kind-test
         name: Build vl3 runner container
         working_directory: /go/src/github.com/cisco-app-networking/nsm-nse/build/ci/runner
         command: |
-          docker build --build-arg vl3_repo=https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}.git --build-arg vl3_branch=${CIRCLE_BRANCH:-master} --build-arg nsm_branch=v0.2.0-vl3 -t vl3-runner:latest -f Dockerfile.runner .
```

Comment:
This is a more open discussion as I wanted to know why we use nsm_branch as some tag (currently we need to update it to v0.2.2-vl3). We are not publishing generated vl3-runner:latest image and I see more sense in testing vl3 scenario with latest code rather than keeping CircleCI stable hiding under some tag. What is your opinion?